### PR TITLE
Update creating-a-pull-request-from-a-fork.md

### LINKED ISSUE #29279: In Creating-a-pull-request-using-fork.md file a particular statement needs an example to understand  for beginners.
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

[Link to the article affected](https://github.com/github/docs/blob/main/content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork.md)

### What part(s) of the article would you like to see updated?

The Statement **If your pull request compares your topic branch with a branch in the upstream repository as the base branch, then your topic branch is also called the compare branch of the pull request** is a bit confusing to comprehend all in one read. Hence, to make it easy to understand a simple example below it would help. 
The example would help beginners more easily understand what is **topic branch, upstream repository, base branch, compare branch**.

### Additional information

I can work on this issue and create a **self-reviewed pull request**

### PR DESCRIPTION
**Pull request by Aviace**
Added a simple example below to easily comprehend the statement "If your pull request compares your topic branch with a branch in the upstream repository as the base branch, then your topic branch is also called the compare branch of the pull request".

### Why: The example would help beginners more easily understand what is "topic branch, upstream repository, base branch, compare branch".

Closes: [Issue-29279](https://github.com/github/docs/issues/29279)

### What's being changed (if available, include any code snippets, screenshots, or gifs):
Screenshot 2023-10-21 103131

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by sophietheking** [comment]
@Aviace 👋 Thanks for opening this PR, I've triaged it, and it will be reviewed shortly. ⚡

**Comment by Aviace** [comment]
> @Aviace 👋 Thanks for opening this PR, I've triaged it, and it will be reviewed shortly. ⚡

Sure!

**Review by Tyszek343 — COMMENTED** [review]
👍👍👍👍

**Comment by Aviace** [comment]
> @Aviace 👋 Thanks for opening this PR, I've triaged it, and it will be reviewed shortly. ⚡

Any updates?

**Review by am-stead — COMMENTED** [review]
👋 @Aviace, 
thanks so much for opening this issue, and for creating an PR for the improvement! 💛

I agree with you that existing content could benefit from a better explanation of the terms involved with examples! 

I love your suggestion to define each term and situate them in relation to one another.💥

I've made two additional commit suggestions, mainly just breaking your useful example down even further into distinct sentences.

File: content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork.md [hunk 1]
```diff
@@ -16,7 +16,15 @@ topics:
...
+If your pull request compares your topic branch with a branch in the upstream repository as the base branch, then your topic branch is also called the "compare branch" of the pull request.
+
+- Your topic branch (also known as “feature branch”) is the branch where you’re making your changes in your forked repository. For example, `my-topic-branch`.
+
+- The base branch is the branch in the upstream (central) repository that you want to merge your changes into. For example, `main`. 
+
+Since the pull request compares the topic branch (`my-topic-branch`) with the base branch (`main`), the topic branch is also known as the “compare branch” of the pull request. 
+
+For more information about pull request branches, including examples, see "[AUTOTITLE](/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request#changing-the-branch-range-and-destination-repository)."
```
Comment (am-stead) [inline content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork.md:27]:
```suggestion
If your pull request compares your topic branch with a branch in the upstream repository as the base branch, then your topic branch is also called the "compare branch" of the pull request.

For example:
- Your topic branch (also known as “feature branch”) is the branch where you’re making your changes in your forked repository (e.g. `my-topic-branch`).
- The base branch is the branch in the upstream (central) repository that you want to merge your changes into (e.g. `main`).
- The pull request compares the changes proposed by the topic branch (`my-topic-branch`) with the base branch (`main`), so `my-topic-branch` is known as the “compare branch”.

For more information about pull request branches, including examples, see "[AUTOTITLE](/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request#changing-the-branch-range-and-destination-repository)."
```

Hoping this commit fixes the failing test (and makes a small tweak to wording)

**Review by cmwilson21 — APPROVED** [review]

**Comment by Aviace** [comment]
Thankyou for letting me learn and contribute to this repository! Even though my first PR wasn't perfect, I learned to make changes and commit to make it meet the standard guidelines. I absolutely love Open Source. Learn-on-the-go.

**Pull request merged**